main-site/components/testimonials-section/testimonial-card/TestimonialCard.styles.ts

```diff
@@ -52,20 +51,20 @@ const StyledTestimonialCardAuthor = styled(
 )<StyledTestimonialCardAuthorYearImageProps>`
   font-size: ${(props): string => (props.isSponsor ? '1.8em' : '1.2em')};
   font-family: ${(props): string =>
-    props.isSponsor ? fonts.nunitoRegular : ''};
+    props.isSponsor ? fonts.nunitoRegular : fonts.nunitoSansRegular};
   margin-bottom: ${(props): string => (props.isSponsor ? '0em' : '0.4em')};
-  color: ${(props): string => (props.isSponsor ? colors.TEXT_BOX : '')};
+  color: ${(props): string => (props.isSponsor ? colors.WHITE : colors.WHITE)};
 `;
 
 const StyledTestimonialCardYearCompany = styled(
   P
 )<StyledTestimonialCardAuthorYearImageProps>`
   font-size: ${(props): string => (props.isSponsor ? '1.2em' : '1em')};
   color: ${(props): string =>
-    props.isSponsor ? colors.BLACK : colors.TEXT_GREY};
+    props.isSponsor ? colors.WHITE : colors.WHITE};
   margin-bottom: 0.4em;
   font-family: ${(props): string =>
-    props.isSponsor ? fonts.nunitoSansSemibold : ''};
+    props.isSponsor ? fonts.nunitoRegular : fonts.nunitoRegular};
```

Comment:
same comment for this and line 64, can you get rid of the conditional if it's always the same